package.json

```diff
@@ -58,6 +58,7 @@
     "pluralize": "^8.0.0",
     "prop-types": "^15.6.0",
     "query-string": "^6.3.0",
+    "rc-tooltip": "^3.7.3",
```

Comment:
any reason we cannot use antd Tooltip but need to install rc-tooltip?

Reply:
i added rc-tooltip because gen3-ui-component uses rc-tooltip, i'm matching the popup style. also, the antd tooltip styling clashes with the navbar tooltips, which have a darker outline, arrow tip, white background, no box shadow. the antd tooltip has box shadow and looks much different. so in healpreprod, where navbar tooltips are enabled, the accessibility tooltip appearance clashes (it's right near the navbar icons)